### ISSUE TITLE
ESGO custom settings

### ISSUE THREAD

**Issue by Chickenudoom** [Lknechtli/BetterCombatEvolved#32]
With permission, I'm hoping we can fork from ESGO and implement our own custom settings.
1. Game Difficulty level changes different enemy type's HP/damage. Customizable sliders would be removed and work strictly through game difficulty.
   Difficulty levels would also determine some Player sliders too, such as damage output for swords, bombs, signs, dots, and crossbows.
   All original ESGO slider would be removed from those values.
2. Set custom attack speed multipliers that are attached to light and heavy sword skills, as well as a speed multiplier for rolling.
   Customizable sliders for these values would be removed and unchangable.
3. Set increased range for rend and whirl on, and remove the option to turn it off.
4. Set custom settings for blocking/parrying, as well as finishers. These sliders would also be removed.
5. Sliders for soft-lock and control options can remain how they currently are in ESGO.
6. Enemy scaling toggle on/off.
7. Level requirements turned off, option removed. (This will require some tweaks to crafting so players don't get OP gear too quickly)
8. Basically only 2 options will be available: enemy scaling on/off and soft-lock on/off (and a slider for KB&M/Gamepad)

We would then need to have 2 versions of BCEv2. One with custom ESGO settings, and one without them (for people who want to install and use ESGO's customization sliders)

**Comment by TriangleAuthority** [comment]
How do you feel about adding a menu separate from ESGO? That way you wouldn't have to merge scaling.xml and dynamicscaling.ws. Things that are already ESGO options could be a preset for ESGO.

**Comment by Chickenudoom** [comment]
That would be good too :) actually would make things a lot less complicated!

**Comment by BardBlue** [comment]
Yeah, best to keep them separate. I would really like it if you guys implemented something to make crafted gear not OP if you have level requirements turned off though. Could be like, crafted gear has -5% damage/armor for each level below the requirement you are, so a weapon 10 levels above you would deal -50% damage or have 50% less armor (numbers to be decided, and there could be a buffer where items up to 5 levels above yours are unaffected.) Since crafted gear is pretty much the only things that you can acquire at way too of a level (especially with level scaling turned off), that'd be pretty much the only stuff affected by this change.

I think you should just give detailed suggested settings for ESGO on your description for settings he already has.

**Comment by TriangleAuthority** [comment]
Hey, just pushed https://github.com/TriangleAuthority/BetterCombatEvolved/compare/master...TriangleAuthority:sliders with a BCEv options menu. all it has is armor stamina cost stuff. remember to move the bin folder to your install folder like with ESGO
